src/Resources/views/Download/creditMemo.html.twig

```diff
@@ -32,7 +32,7 @@
                             <img src="https://demo.sylius.com/assets/shop/img/logo.png" style="width:300px;">
                         </td>
                         <td>
-                            {{ 'sylius_refund.ui.issued_at'|trans }}: {{ creditMemo.issuedAt|date('Y-m-d H:i:s') }}<br>
+                            {{ 'sylius_refund.ui.issued_at'|trans([], 'messages', creditMemo.localeCode) }}: {{ creditMemo.issuedAt|date('Y-m-d H:i:s') }}<br>
```

Comment:
What do you think about extraction this `creditMemo.localeCode` to a variable, instead of copying it 17 times :D?

Reply:
i dont think if it makes any difference :D this way we know every translation gets credit memo locale code.